src/components/button/Button.jsx

```diff
@@ -0,0 +1,27 @@
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import "src/components/button/Button.component.scss";
+
+export const Button = ({
+  onButtonClick,
+  buttonText,
+  srcIcon,
+  altText,
+  buttonClass,
```

Comment:
```suggestion
  buttonText,
  srcIcon,
  altText,
  buttonClass,
  onButtonClick,
```

https://github.com/azo-wtag/todo-app-react/pull/3#discussion_r1172165121